pydis_site/settings.py

```diff
@@ -278,9 +294,18 @@
 
 # Custom settings for django-simple-bulma
 BULMA_SETTINGS = {
-    "variables": {
-        "green": "#21c65c",  # Accessibility: Better contrast with the light text
-        "primary": "#7289DA",
+    "variables": {  # If you update these colours, please update the notification.css file
+        "primary": "#7289DA",  # Discord blurple
+
+        "orange": "#ffb39b",  # Bulma default, but at a saturation of 100
+        "yellow": "#ffea9b",  # Bulma default, but at a saturation of 100
```

Comment:
can we line these up?

Reply:
Not sure what you mean by lining up here either

Reply:
```py
thing()  # this
with.fire()  # looks
pull_chain()  # messy

thing()       # this
with.fire()   # looks
pull_chain()  # clean
```